monitoring-samples/stream-apps/README.md

```diff
@@ -1,37 +1,41 @@
 # Stream Apps Monitoring
 
-Ensure that the POM contains:
+Add `org.springframework.cloud:spring-cloud-stream` and `org.springframework.cloud:spring-cloud-starter-stream-rabbit` to your Spring Boot application 
+POM to turn it into Stream Cloud Stream Application:
 
 ```xml
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-starter-stream-rabbit</artifactId>
-		</dependency>
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-stream</artifactId>
+    </dependency>
+    
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+    </dependency>
+```
 
+To replace Rabbit by Kafka binder use the `org.springframework.cloud:spring-cloud-starter-stream-kafka` dependency instead instead.
 
-		<dependency>
-			<groupId>org.springframework.cloud.stream.app</groupId>
-			<artifactId>app-starters-micrometer-common</artifactId>
-			<version>2.1.2.RELEASE</version>
-		</dependency>
-		<dependency>
-			<groupId>io.micrometer</groupId>
-			<artifactId>micrometer-registry-prometheus</artifactId>
-		</dependency>
+To enable SCDF Prometheus Proxy monitoring for your application you would need to add the following additional additional dependencies: 
 
-		<dependency>
-			<groupId>io.micrometer.prometheus</groupId>
-			<artifactId>prometheus-rsocket-spring</artifactId>
-			<version>0.9.0</version>
-		</dependency>
-		...
-	</dependencies>
+```xml
+    <dependency>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>app-starters-micrometer-common</artifactId>
```

Comment:
The `app-starters-micrometer-common` purpose is still not described. Perhaps a one-liner about what this adds to the applications would help. Esp, this is a required dependency if users would have to use the monitoring support in SCDfF.

Reply:
Oh, nevermind! You have it described below.